ocpmodels/datasets/lmdb_dataset.py

```diff
@@ -117,12 +135,44 @@ def __getitem__(self, idx):
             data_object = pyg2_data_transform(pickle.loads(datapoint_pickled))
             data_object.id = f"{db_idx}_{el_idx}"
         else:
-            datapoint_pickled = self.env.begin().get(self._keys[idx])
+            datapoint_pickled = self.env.begin().get(
+                f"{self._keys[idx]}".encode("ascii")
+            )
             data_object = pyg2_data_transform(pickle.loads(datapoint_pickled))
 
         if self.transform is not None:
             data_object = self.transform(data_object)
 
+        if "augment" in self.config:
+            if "positions" in self.config["augment"]:
+                # randomly rattle the positions
+                data_object.pos += torch.Tensor(
+                    np.random.normal(
+                        loc=0,
+                        scale=self.config["augment"]["positions"],
+                        size=data_object.pos.shape,
+                    )
+                )
+            if "isotropic_strain" in self.config["augment"]:
+                # isotropic strain by percentage
+                data_object.cell *= torch.Tensor(
+                    [
+                        np.random.normal(
+                            loc=1,
+                            scale=self.config["augment"]["isotropic_strain"],
+                        )
+                    ]
+                )
+            if "anisotropic_strain" in self.config["augment"]:
+                # isotropic strain by percentage
+                data_object.cell *= torch.Tensor(
+                    np.random.normal(
+                        loc=1,
+                        scale=self.config["augment"]["anisotropic_strain"],
+                        size=data_object.cell.shape,
+                    )
+                )
+
```

Comment:
- This augmentation layer should ideally be somewhere else independent of the data backend that's used. Maybe as part of the trainer before doing a forward pass through the model.
- Instead of casting from numpy to torch, we can probably [initialize Torch.tensors directly](https://pytorch.org/docs/stable/generated/torch.normal.html)?

Reply:
Removed this from the PR